apps/namadillo/package.json

```diff
@@ -10,7 +10,7 @@
     "@chain-registry/client": "^1.53.5",
     "@cosmjs/encoding": "^0.32.3",
     "@keplr-wallet/types": "^0.12.136",
-    "@namada/chain-registry": "^1.0.0",
+    "@namada/chain-registry": "github:anoma/namada-chain-registry#5a884bf6343f5130c4a419901a87ea86df53dd83",
```

Comment:
quick question: Should we go ahead and merge the release branch on the registry, once everything we need is in place? Then we can just update this to that tag

Reply:
good idea!